package.json

```diff
@@ -121,9 +113,9 @@
     "mt-a11y-dialog": "^5.1.1",
     "prop-types": "^15.6.2",
     "querystringify": "^2.0.0",
-    "react": "^18.0.0",
+    "react": "^16.14.0",
```

Comment:
There has been issues with the latest version of WP. Are we updating to React 18 or keeping it on the old version

Reply:
Same for WordPress/Scripts. Should it be included?

Reply:
I've been holding off on the update until the blocks issues are resolved.
I expect that before this goes out...
I don't want to accidentally create more issues 😉

Reply:
As far as `@wordpress/scripts` - we are aiming to stop including them (any WordPress libraries) manually and just depend on the existing ones from the WP install.